robotino_controller_configuration_gazebo/package.xml

```diff
@@ -13,6 +13,8 @@
   <build_depend>roslaunch</build_depend>
   <build_depend>rostopic</build_depend>
   <build_depend>gazebo_ros_control</build_depend>
+  <build_depend>gazebo-plugins</build_depend>
```

Comment:
I think this package doesn't require the dependencies.
`robotino_description` depends on gazebo_plugins and `robotino_gazebo_worlds` on gazebo_ros, but `robotino_controller_configuration_gazebo` only contains ros controllers configuration and the nodes related to the controls.